tutor/src/screens/assignment-edit/builder.js

```diff
@@ -63,15 +63,31 @@ const HintText = styled.div`
   line-height: 2rem;
 `;
 
+const TextInputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TextInputError = styled.div`
+  color: red;
+  margin: 1rem 0;
+`;
+
 const StyledTextInput = styled(Field).attrs({
   type: 'text',
 })`
   padding: 0.8rem 1rem;
   border-radius: 4px;
   border: 1px solid ${colors.forms.borders.light};
-  font-size: 1.4rem;
-
+  font-size: 1.2rem;
+  /** styling errors when template name is invalid */
+  background: ${props => props.haserror ? '#fbe8eA' : '#FFFFFF'};
+  color: ${props => props.haserror ? 'red' : 'black'};
+  border-color: ${props => props.haserror ? '#f4c0c5' : 'd5d5d5'};
+  border-width: ${props => props.haserror ? '2px' : '1px'};
+  
```

Comment:
rather than using hex values here, how about adding them to the `colors.states` in theme.js? seems like something that should be standardized.